### PR TITLE
Martin/single stack fixes

### PR DESCRIPTION
**Pull request by MartinJibber**
Main points:
Removing the members vc from the header.
Removing the decoration view that showed unread message count.
Adjusting the message layout so it works better with the single stack.

### CODE REVIEW

File: Shared/Flows/Onboarding/Welcome/WelcomeCollectionView.swift [hunk 1]
```diff
@@ -17,7 +17,6 @@ class WelcomeCollectionView: CollectionView {
 
     init() {
         let layout = MessagesTimeMachineCollectionViewLayout()
-        layout.hideCenterDecorationView = true
```
Comment (MartinJibber) [inline Shared/Flows/Onboarding/Welcome/WelcomeCollectionView.swift:20]: Removing the decoration view.

File: Shared/UI/Views/MessageContentView/MessageContentView.swift [hunk 2]
```diff
@@ -103,25 +103,28 @@ class MessageContentView: BaseView {
         self.authorView.size = self.getAuthorSize()
         self.authorView.pin(.top, offset: MessageContentView.padding)
         self.authorView.pin(.left, offset: MessageContentView.padding)
-        self.authorView.centerOnY()
```
Comment (MartinJibber) [inline Shared/UI/Views/MessageContentView/MessageContentView.swift:106]: Author view is pinned to the top left.

File: iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift [hunk 3]
```diff
@@ -171,13 +171,9 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
 
         switch state {
         case .read:
-            self.collectionLayout.spacingKeyPoints = [0, 40, 74, 86]
+            self.collectionLayout.spacingKeyPoints = [0, 96, 144, 192]
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift:174]: Adjusting spacing in the read mode.

File: Shared/UI/Views/MessageContentView/MessageContentView.swift [hunk 4]
```diff
@@ -103,25 +103,28 @@ class MessageContentView: BaseView {
...
         self.textView.size = self.textView.getSize(width: self.bubbleView.bubbleFrame.width)
-
-        self.textView.textAlignment = .left
         self.textView.match(.left, to: .right, of: self.authorView, offset: MessageContentView.padding)
-        self.textView.centerOnY()
+        if self.textView.numberOfLines > 2 {
+            self.textView.pin(.top, offset: MessageContentView.padding)
+            self.textView.textAlignment = .left
+        } else {
+            self.textView.centerOnY()
```
Comment (Benjamindodgson) [inline Shared/UI/Views/MessageContentView/MessageContentView.swift]: Lets keep the the text in the same position regardless of line count

**Review by Benjamindodgson — CHANGES_REQUESTED** [review]
I think we should keep the members classes for now as they will be reused elsewhere. Lets also keep the text position consistent now that we have the avatar always present.

**Pull request merged**